src/meta_utils.py

```diff
@@ -0,0 +1,528 @@
+import numpy as np
+import pandas as pd
+from typing import Optional, Tuple, Callable
+
+# (1) Crop land **mapping** <- MOST GENERAL
+#     -> NOTE: With crop land map there is no 'final' agreement between two labeler 
+#              sets b/c there is typically no *forced* agreement or resolvement.
+
+# (2) Crop land **area estimation**
+#     -> NOTE: With area estimation there *is* final agreement between the two labeler sets. <- MOST COMMON
+#     -> NOTE: Additionally; area estimation may also be for either single year (map) or 
+#              multi-year (area change).
+
+# (3) With area estimation there are additionally two types:
+#     -> Single-year crop map area estimation
+#     -> Multi-year crop map change area estimation 
+
+# (3) Difference in **mapping** and **area estimation**
+#     -> mapping : two csv files (set 1, set 2)
+#     -> area est. : three csv files (set 1, set 2, 'final')
+
+# (4) Goal:
+#     -> Generalize functions st behavior adjusted depending on if labeling project is **mapping** 
+#        or **area estimation** 
+#     -> Don't require additional script file; instead have two separate notebooks for mapping
+#        and area estimation but all util functions in one .py
+
+def check_dataframes(
+        df1 : pd.DataFrame, 
+        df2 : pd.DataFrame, 
+        df3 : Optional[pd.DataFrame] = None
+    ) -> Tuple[pd.DataFrame, ...]:
+    """ Performs checks on labeling CSVs loaded to dataframes. """
+
+    if df3 is not None:
```

Comment:
This code looks fine functionally! 
A bit of a nit, but the if statement goes against the DRY principle (Don't repeat yourself). There's quite a lot of repetition in the if and else block. Practically I think 99% of our data will have at most two dataframes so having code to handle only two for now would be more than fine. Theoretically you could have a `dfs` list as an input but here I'll invoke another software principle [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it#:~:text=%22You%20aren't%20gonna%20need,need%20it%22%20(YAGNI).) (you aren't gonna need it) :D